subprojects/c-flags.wrap

```diff
@@ -0,0 +1,8 @@
+[wrap-file]
+directory = c-flags-1.5.3
+source_url = https://github.com/DieTime/c-flags/archive/1.5.3.tar.gz
+source_filename = c-flags-1.5.3.tar.gz
+source_hash = 81dd6d961c46f0039c58a381d2cb2aba786d553f02271f39d8b34722a2c4689d
+
+[provide]
+libcflags = libcflags_dep
```

Comment:
Hmm... the dependency which you provide as a wrap interface is "libcflags" but the pkg-config file installed is "c-flags.pc", so this doesn't provide the same interface. The wrap dependency name should really correspond to the name that `dependency()` would detect via pkg-config.

Reply:
Do I understand correctly that the dependency name `c_flags` will be correct in case of `c-flags.pc`?

Reply:
No, but `c-flags` will be correct in the case of `c-flags.pc`. :)

Reply:
Thanks, I fixed the mistake.

Reply:
Also corrected `releases.json`

Reply:
In version 1.5.4 corrected the export of symbols.
Pull Request: DieTime/c-flags#2.